.vscode/extensions.json

```diff
@@ -0,0 +1,3 @@
+{
+  "recommendations": ["dbaeumer.vscode-eslint", "esbenp.prettier-vscode"]
```

Comment:
I am really hoping these extensions don't become vulnerable, thus affecting all cdxgen users. Do we really need this .vscode folder, or could we keep it optional under `contrib` directory?

Reply:
Removed from this PR